charts/gatekeeper/templates/gatekeeper-mutating-webhook-configuration-mutatingwebhookconfiguration.yaml

```diff
@@ -43,7 +43,7 @@ webhooks:
       - {{ . }}
       {{- end }}
     {{- end }}
-  objectSelector: {{ toYaml .Values.mutatingWebhookObjectSelector }}
```

Comment:
no need to modify `/charts`, this will be automatically updated as part of release. you can revert any changes to `charts`.

Reply:
resolved :) thank you